index.js

```diff
@@ -55,6 +56,7 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
     if (path.sep === '\\' && typeof string === 'string') {
       altString = string.split('\\').join('/');
```

Comment:
Although I'm not sure I need to address those edge cases. split/join is more performant than regex,  isn't it? And why would a windows user ever supply a path with a leading backslash?